osc-server/src/main/java/org/osc/core/broker/service/persistence/SecurityGroupInterfaceEntityMgr.java

```diff
@@ -50,8 +53,15 @@ public static void fromEntity(SecurityGroupInterface sgi, SecurityGroupInterface
         dto.setParentId(sgi.getVirtualSystem().getId());
         dto.setName(sgi.getName());
         dto.setTagValue(sgi.getTagValue());
-        dto.setPolicyId(sgi.getMgrPolicy() != null ? sgi.getMgrPolicy().getId() : null);
-        dto.setPolicyName(sgi.getMgrPolicy() != null ? sgi.getMgrPolicy().getName() : null);
+		if (!sgi.getPolicies().isEmpty()) {
+			Set<PolicyDto> policySet = new HashSet<>();
+			for (Policy policy : sgi.getPolicies()) {
+				PolicyDto policyDto = new PolicyDto();
+				PolicyEntityMgr.fromEntity(policy, policyDto);
+				policySet.add(policyDto);
+			}
+			dto.setPolicies(policySet);
```

Comment:
Ensure that the dto.policies  has an empty set rather than null when the sgi.getpols.isempty

Reply:
done